pkg/util/provider/machinecontroller/controller.go

```diff
@@ -19,6 +19,7 @@ package controller
 
 import (
 	"fmt"
+	"github.com/gardener/machine-controller-manager/pkg/util/worker"
```

Comment:
pls keep this import in the desired location as explained in comment above

Reply:
Yes, now it follows the scheme: stdlib imports, repo imports and then third party imports.